app/js/utils.js

```diff
@@ -70,6 +165,25 @@
         metadata: metas
       };
     },
+
+    meta_to_human: function(metas) {
```

Comment:
nit: maybe a name like `parse_meta` is better. i'd have a better idea of what it was doing then.